feat: Add boolean (#15)
thisdot/tech-bootcamps: README.md

Bloom Institute of Technology is a online tech school. It offers Courses on Web3 Development, Backened Development, Data Science and Full Stack Development.

- [Boolean](https://boolean.careers/)

> Boolean is an international tech academy and with the following two courses: Full-Stack Web Development and Data Analytics. Both courses have a duration of 6 months.

Reviewer: ```suggestion
Boolean is an international tech academy that offers Full-Stack Web Development and Data Analytics courses. Both courses have a duration of 6 months.
```
